src/common/Asio/IpAddress.h

```diff
@@ -28,7 +28,7 @@ namespace Trinity
 #if BOOST_VERSION >= 106600
         using boost::asio::ip::make_address;
         using boost::asio::ip::make_address_v4;
-        inline uint32 address_to_uint(boost::asio::ip::address_v4 const& address) { return address.to_uint(); }
+        inline uint64 address_to_uint(boost::asio::ip::address_v4 const& address) { return address.to_uint(); }
```

Comment:
why was this changed? ipv4 address always fits into uint32

Reply:
Based on the csv file, exceeds the value uint32:

```
3758096128,"3758096383","AU","Australia"
```

Reply:
No, its impossible, ipv4 was designed to use exactly 4 bytes
also, 3758096128 < 4294967295 (it fits)